ansible/ansible.cfg

```diff
@@ -1,5 +1,6 @@
 [defaults]
 ansible_managed = Ansible managed, do not edit directly
+role_path = /opt/ansible_roles
 
 # Added to allow HQ to be cloned as a non-privileged user in Ansible >2.0.
 allow_world_readable_tmpfiles = True
```

Comment:
+1 for clone as root and then `chown cchq`. It seems better to not have this enabled as we're developing new code because we may inadvertently end up depending on it in places where we should not.